client/starwhale/api/_impl/wrapper.py

```diff
@@ -109,7 +117,10 @@ def log_result(
         record = {"id": data_id, "result": _serialize(result) if serialize else result}
         for k, v in kwargs.items():
             record[k.lower()] = _serialize(v) if serialize else v
-        self._log(self._results_table_name, record)
+
+        self._log(
```

Comment:
`Evaluation` Class override `_log` and `_flush` method, add add `self._fetch_table_name` logical?

```
class Evalution(Logger):

def _log(self, table_name: str, record: Dict[str, Any]) -> None:
    table_name = self._fetch_tablename(table_name)
    super()._log(table_name, record)
```

Reply:
fixed